docs/whats-new/release-notes/v2_14_0.md

```diff
@@ -47,18 +48,36 @@ Zowe Version 2.14.0 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
+- Corrected extra character being displayed at the end of lines when issuing command `zowe files compare` on Windows. [#1992](https://github.com/zowe/zowe-cli/issues/1992)
+- Corrected the online help description for `zowe files compare uss`. [#1754](https://github.com/zowe/zowe-cli/issues/1754)
+- Fixed typo in command help for `zowe zos-workflows create` commands.
+
 #### Zowe CLI Imperative Framework
 
-#### CICS Plug-in for Zowe CLI
+- Fixed error message shown for null option definition to include details about which command caused the error. [#2002](https://github.com/zowe/zowe-cli/issues/2002)
```

Comment:
@adam-wolfe -- would like to add more detail here. for instance, where is the null option definition done?

Reply:
Option definitions for a command are specified in the Zowe CLI source code (or in Zowe CLI plug-ins) in each command's CommandDefinition. The options are specified as an array of OptionDefinitions. 

If a developer happened to add a null or undefined variable to the options array of a CommandDefinition then build and run Zowe CLI, the CLI used to fail with an unhelpful error message that would print out the entire command tree. With this PR, the error message has been revised to display the command handler file path for the command that failed, which makes it easier for a developer to figure out which CommandDefinition is broken.

Reply:
I updated the note. Please review for accuracy.

Reply:
Looks good